Makefile

```diff
@@ -62,6 +66,16 @@ ifneq ($(OVERRIDE_IMAGE_NAME),)
 	sudo docker tag $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) $(OVERRIDE_IMAGE_NAME)
 endif
 
+#This rule need to be run on Windows 19 server with golang, cygwin, mktemp and docker installed.
+container_win: build
```

Comment:
Go allows you to cross-compile (i.e build `.exe` on Mac etc). We should add a `build_windows` target in the Makefile that emits a `.exe` file as relevant and depend on that for this target. Along the lines of:

```
GOOS=windows GOARCH=amd64 go build -ldflags "$(LDFLAGS)" -o $(OUT_DIR)/$(ARCH)/$(BINARY_NAME) ./cmd/wavefront-collector/
```

Not familiar with Windows container images. Can the docker image be built on a Mac (using `docker build`) with this Dockerfile?

That'll simplify the build side of things.

Reply:
Windows docker images have to be built on a Windows machine.